### PR TITLE
Fix Static Content CI Push

### PR DESCRIPTION
**Pull request by brunobowden**
- Standardize on projectID usage
- Correct project id
- Remove deprecated code in push script

## How did you test the change?

CI Push Static Content after merge

## Checklist:

- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

**Comment by brunobowden** [comment]
FYI @advayDev1 @theswerd

File: tools/build-and-push-static-serving.sh [hunk 1]
```diff
@@ -12,17 +12,13 @@ if [ "$DONT_BUILD" != "true" ]; then
...
     exit 1
   fi
-  # TODO: remove once .github/workflows/static-content.yaml is updated
-  if [[ $BUCKET == *static-content* ]]; then
-    echo "Deprecated argument for bucket name: $BUCKET"
-  else
-    BUCKET=$BUCKET-static-content
-  fi
+
+  PROJECT_ID=$PROJECT_ID-static-content
```
Comment (matthewblain) [inline tools/build-and-push-static-serving.sh:21]: BUCKET?

**Review by matthewblain — COMMENTED** [review]
these ids are in so many places!

File: tools/build-and-push-static-serving.sh (hunk 1 above)
Comment (brunobowden) [reply tools/build-and-push-static-serving.sh:21]: Good catch. Fixed.

**Comment by brunobowden** [comment]
@matthewblain - I agree about it being repeated in far too many places. Should be good now.

**Review by matthewblain — APPROVED** [review]

**Pull request merged**